lib/action_cable/helpers/websocket_url_helper.rb

```diff
@@ -0,0 +1,29 @@
+module ActionCable
+  module Helpers
+    module WebsocketUrlHelper
+      # Returns a "websocket-url" meta tag with the value of the url specified in your
+      # configuration. Ensure this is above your javascript tag:
+      #
+      #   <head>
+      #     <%= websocket_url_meta_tag %>
+      #     <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+      #   </head>
+      #
+      # This is then used by ActionCable to determine the url of your websocket server.
+      # Your CoffeeScript can then connect to the server without needing to specify the
+      # url directly:
+      #
+      #   #= require cable
+      #   @App = {}
+      #   App.cable = Cable.createConsumer()
+      #
+      # Make sure to specify the correct server location in each of your environments
+      # config file:
+      #
+      #   config.action_cable.url = "ws://example.com:28080"
+      def websocket_url_meta_tag
+        tag "meta", name: "websocket-url", content: Rails.application.config.action_cable.url
+      end
```

Comment:
Would name these methods to reflect that it's for Action Cable, not a generic "websocket."